### DISCUSSION TITLE
Webinar Recording: CI/CD workflows for your docs 🤖

### DISCUSSION THREAD

**Discussion by addisonschultz** [GitbookIO/community#729] [Announcements]
Hey GitBookers! We've got a new recording for you — Last night's webinar with Mike from Moderne! This was a great webinar that dove in the behind the scenes workings of how CI/CD workflows can work with GitBook and GitSync to automate different parts of your docs workflows. 

Do things like automate tests, spellchecks, and much more.

Check it out here! 🤩 

https://www.gitbook.com/events/gitbook-moderne-september

**Comment by abigailrascon9** [comment]
prepare for cissp exam with edusum's practice exam. https://www.edusum.com/isc2/cissp-isc2-information-systems-security-professional

**Reply by abigailrascon9** [reply]
prepare for cissp exam with edusum's practice exam. https://www.edusum.com/isc2/cissp-isc2-information-systems-security-professional

**Comment by abigailrascon9** [comment]
prepare for cissp exam with edusum's practice exam. https://www.edusum.com/isc2/cissp-isc2-information-systems-security-professional
